main/src/ca/uwaterloo/flix/language/ast/SimplifiedAst.scala

```diff
@@ -264,6 +264,22 @@ object SimplifiedAst {
 
   }
 
+  case class LabeledExpression(label: Int, exp : Expression) extends Expression {
```

Comment:
This class does not belong here, because it is only used by the CFA phase. So move it there.